lapack/testlapack/dtrtrs.go

```diff
@@ -0,0 +1,140 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/lapack"
+
+	"gonum.org/v1/gonum/blas/blas64"
+
+	"gonum.org/v1/gonum/floats"
+
+	"golang.org/x/exp/rand"
+	"gonum.org/v1/gonum/blas"
```

Comment:
Sometimes goimports doesn't do quite the right thing.

```
	"golang.org/x/exp/rand"

	"gonum.org/v1/gonum/blas"
	"gonum.org/v1/gonum/blas/blas64"
	"gonum.org/v1/gonum/floats"
	"gonum.org/v1/gonum/lapack"
```

Reply:
Sorry, didn't notice that. Amended.